cmd/edenStart.go

```diff
@@ -98,11 +102,20 @@ var startCmd = &cobra.Command{
 		if eveRemote {
 			return
 		}
-		if err := eden.StartEVEQemu(command, configName, qemuARCH, qemuOS, eveImageFile, qemuSMBIOSSerial, qemuAccel, qemuConfigFile, eveLogFile, evePidFile); err != nil {
-			log.Errorf("cannot start eve: %s", err)
+		if eveParallels {
```

Comment:
You can see the example of logic for devModel here: https://github.com/lf-edge/eden/blob/97877c68d13df9218f6ee798f55eb4aa9f2f742b/cmd/edenSetup.go#L159